demo/src/screens/incubatorScreens/WheelPickerScreen.tsx

```diff
@@ -39,75 +34,33 @@ const useData = (initialMonth?: string, initialYear?: string, initialDays?: numb
     setShowDialog(false);
   }, []);
 
-  const onMonthChange = useCallback((item: WheelPickerValue, _: number) => {
-    setMonth(item);
-  }, []);
-
-  const onYearChange = useCallback((item: WheelPickerValue, _: number) => {
-    setYear(item);
-  }, []);
-
-  const onDaysChange = useCallback((item: WheelPickerValue, _: number) => {
-    setDays(item);
-  }, []);
-
-  return {
-    onMonthChange,
-    onYearChange,
-    onDaysChange,
-    selectedMonth,
-    selectedDays,
-    onPickDaysPress,
-    onDialogDismissed,
-    showDialog
-  };
-};
-
-export default () => {
-  const {
-    selectedMonth,
-    onMonthChange,
-    onYearChange,
-    selectedDays,
-    onDaysChange,
-    onPickDaysPress,
-    onDialogDismissed,
-    showDialog
-  } = useData('February', undefined, 5);
-
   return (
     <View flex padding-page>
       <Text h1>Wheel Picker</Text>
 
       <View marginT-s5 centerH>
         <Text h3>Months</Text>
         <Incubator.WheelPicker
-          onChange={onMonthChange}
+          initialValue={'February'}
           activeTextColor={Colors.primary}
           inactiveTextColor={Colors.grey20}
           items={monthItems}
           textStyle={Typography.text60R}
-          selectedValue={selectedMonth}
         />
 
         <Text h3>Years</Text>
         <Text bodySmall grey30>
           (Uncontrolled, initialValue passed)
```

Comment:
Why do we need this example? Looks like the days example so we don't really need both now, no?

Reply:
I would add an example for changing 'initialValue' to show its effect, like a button that rolls the wheel by changing the prop's value.

Reply:
Not sure why we had the months example originally, maybe for string values.
Anyway, I've added two buttons for updating the years value with the `initialValue`.